lib/puppet/type/exec.rb

```diff
@@ -31,19 +45,19 @@ module Puppet
       is non-standard, and can be affected by the `refresh` and
       `refreshonly` attributes:
 
-      * If `refreshonly` is set to true, the exec will _only_ run when it receives an
+      * If `refreshonly` is set to true, the exec runs _only_ when it receives an
         event. This is the most reliable way to use refresh with execs.
-      * If the exec already would have run and receives an event, it will run its
+      * If the exec  would have already run and receives an event, it runs its
         command **up to two times.** (If an `onlyif`, `unless`, or `creates` condition
         is no longer met after the first run, the second run will not occur.)
```

Comment:
I'm fuzzy on "would have already run"? Is that the case where "refreshonly" is set to false (default), and the exec subscribes to another resource. In which case the exec might run twice?

Reply:
Honestly, I only changed the location of the word "already" to the correct place, so I'm not completely sure. All of these bullet points seem a bit fuzzy to me, so I'll give them a once over.